x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -59,46 +66,89 @@ const nodeAssets = {
   },
 };
 
-const ChildEventsButton = React.memo(() => {
-  return (
-    <EuiButton
-      onClick={useCallback((clickEvent: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
-        clickEvent.preventDefault();
-        clickEvent.stopPropagation();
-      }, [])}
-      color="ghost"
-      size="s"
-      iconType="arrowDown"
-      iconSide="right"
-      tabIndex={-1}
-    >
-      {i18n.translate('xpack.endpoint.resolver.relatedEvents', {
-        defaultMessage: 'Events',
-      })}
-    </EuiButton>
-  );
-});
-
-const RelatedAlertsButton = React.memo(() => {
-  return (
-    <EuiButton
-      onClick={useCallback((clickEvent: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
-        clickEvent.preventDefault();
-        clickEvent.stopPropagation();
-      }, [])}
-      color="ghost"
-      size="s"
-      tabIndex={-1}
-    >
-      {i18n.translate('xpack.endpoint.resolver.relatedAlerts', {
-        defaultMessage: 'Related Alerts',
-      })}
-    </EuiButton>
-  );
-});
+const eventTypeToNameMap = new Map<string, string>([
```

Comment:
These should not be the phase 0 event types, and instead should be event.catergory 1 and 2 columns from https://docs.google.com/spreadsheets/d/1awkgWQR4e2GWaHYhYPhdwcXYri_CbxRmtl7Ghul-5wU/

Reply:
Would it be a valid approach to just add those things (and their translations) to this Map to cover as much input as possible?

Reply:
I attempted to be as exhaustive as possible here to satisfy the concern: https://github.com/elastic/kibana/pull/64121/commits/d08f4144a7d572b49dc40804cc964b6b41a22310
Is that OK?